ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot

```diff
@@ -83,23 +83,28 @@ Verify Ods Users Can Create And Run A Data Science Pipeline With Ray Using The k
 # robocop: disable:line-too-long
 End To End Pipeline Workflow Using Kfp
     [Documentation]    Create, run and double check the pipeline result using Kfp python package. In the end,
-    ...    clean the pipeline resources.
+    ...    clean the pipeline resources. This keyword will assign Edit permission to the user that was sent as
+    ...    parameter and that user will do the API call.
     [Arguments]    ${username}    ${password}    ${project}    ${python_file}    ${method_name}
     ...    ${pipeline_params}    ${status_check_timeout}=160    ${ray}=${FALSE}
     Remove Pipeline Project    ${project}
     New Project    ${project}
+    Run Command    oc label namespace ${project} opendatahub.io/dashboard=true
     Install DataSciencePipelinesApplication CR    ${project}
-    ${status}    Login And Wait Dsp Route    ${username}    ${password}    ${project}
+    ${status}    Login And Wait Dsp Route    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${project}
```

Comment:
In my opinion, mixing global variables and parameters here for username and password is confusing, it looks like an error. 

I think it would be better to add new parameters  like ${username_create_run}  and ${password_create_run}, or just  not using the keyword `End To End Pipeline Workflow Using Kfp` for this specific test

Reply:
The current approach looks good to me. Thanks Diego!